overlay/frameworks/base/core/res/res/values/config.xml

```diff
@@ -217,7 +217,7 @@
          Wifi interfaces.  If the device doesn't want to support tethering over Wifi this
          should be empty.  An example would be "softap.*" -->
     <string-array translatable="false" name="config_tether_wifi_regexs">
-        <item>"wlan0"</item>
+        <item>"wlp1s0"</item>
```

Comment:
does this fix anything? :)

Reply:
No sure :), just to be cleaner as the interface in utouch is wlp1s0